#! /bin/sh /usr/share/dpatch/dpatch-run
## DP: 02_SVN_STREAM_CHUNK_SIZE_int.dpatch by Emilio Pozuelo Monfort <pochu@debian.org>
## DP: http://bugs.debian.org/712383
##
## All lines beginning with `## DP:' are a description of the patch.

@DPATCH@

diff -ruNp svnmailer-1.0.8.orig/src/lib/svnmailer/subversion.py svnmailer-1.0.8/src/lib/svnmailer/subversion.py
--- svnmailer-1.0.8.orig/src/lib/svnmailer/subversion.py	2006-04-17 12:29:06.000000000 +0200
+++ svnmailer-1.0.8/src/lib/svnmailer/subversion.py	2013-06-15 16:07:36.137576163 +0200
@@ -355,7 +355,7 @@ class Repository(object):
             try:
                 while True:
                     chunk = svn_core.svn_stream_read(
-                        stream, svn_core.SVN_STREAM_CHUNK_SIZE
+                        stream, int(svn_core.SVN_STREAM_CHUNK_SIZE)
                     )
                     if not chunk:
                         break
